account_operating_unit/README.rst

```diff
@@ -0,0 +1,109 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
+
+===============================
+Accounting with Operating Units
+===============================
+
+This module introduces the following features:
+
+* Adds the Operating Unit (OU) to the account move line.
+
+* Defines if the Operating Units are self-balanced and Inter-Operating Unit
+  clearing account at company level.
+
+* Journal entry with lines in different Operating Units are checked based on
+  the "self-balanced" set up in OU.
+
+  At the time of posting the journal entry, the corresponding lines in the
+  Inter-Operating Unit clearing account are automatically created, making
+  each OU self-balanced.
+
+* The account financial reports include the option to filter by OU.
+
+* Adds the Operating Unit (OU) to the invoice.
+
+* Implements security rules in the invoice based on OU.
+
+Installation
+============
+
+No specific installation requirements.
+
+Configuration
+=============
+
+If your company is required to generate a balanced balance sheet by
+Operating Unit you can specify at company level that Operating Units should
+be self-balanced, and then indicate a self-balancing clearing account.
+
+1. Create an account for "Inter-OU Clearing" of type Non-current assets.
+
+2. Go to *Settings / Companies / Configuration* and Set the "Operating Units
+   are self-balanced" checkbox.
+
+   Then set the "Inter-OU Clearing"  account in "Inter-Operating Unit
+   clearing account" field.
+
+3. Assign Operating Unit in Accounts.
+
+
+Usage
+=====
+
+* Add the Operating Unit to invoices.
+
+* Add the Default Operating Unit to account move. Then all move lines will
+  by default adopt this Operating Unit.
+
+* Add Operating Units to the move lines.
+
+  If they differ across lines of the same move, and the OU's are
+  self-balanced, then additional move lines will be created so as to make
+  the move self-balanced from OU perspective.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/213/9.0
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/213/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
```

Comment:
Actually the user can set an account of any type for inter operating unit account. In readme/configuration "Non-current assets" is recommended. What do accounting experts suggest?